CMakeLists.txt

```diff
@@ -58,8 +58,11 @@ if(MSVC)
             # roughly -Wno-sign-compare
```

Comment:
I think `/wd4996` `/wd4722` may be unnecessary as loguru has been deleted for a long time... Can you do me a favor to check if that is the case?

Reply:
/wd4996 is still needed, 4772 warning does't appear under MSVC 2017.

Reply:
Your can check the logs here, https://ci.appveyor.com/project/DsoTsin/ccls/builds/20224202